integrated_channels/blackboard/models.py

```diff
@@ -88,3 +94,93 @@ def get_content_metadata_exporter(self, user):
 
     def get_content_metadata_transmitter(self):
         return BlackboardContentMetadataTransmitter(self)
+
+    def get_learner_data_exporter(self, user):
+        return BlackboardLearnerExporter(user, self)
+
+    def get_learner_data_transmitter(self):
+        return BlackboardLearnerTransmitter(self)
+
+
+@python_2_unicode_compatible
+class BlackboardLearnerDataTransmissionAudit(models.Model):
+    """
+    The payload we send to Blackboard at a given point in time for an enterprise course enrollment.
+
+    """
+    blackboard_user_email = models.EmailField(
+        max_length=255,
+        blank=False,
+        null=False,
+        help_text='The learner`s Blackboard email. This must match the email on edX'
+    )
+
+    enterprise_course_enrollment_id = models.PositiveIntegerField(blank=False, null=False, db_index=True)
```

Comment:
this is a total nit but just for easier field browsing, lay them out alphabetically here?